src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/security/authentication/GithubAuthenticator.kt

```diff
@@ -115,7 +103,7 @@ class GithubAuthenticator(private val userRepository: UserRepository,
                 ?: throw BadConfigurationError("GitHub org $githubOrg has no team called $teamName")
 
         val members = teamService.getMembers(team.id)
-        return members.contains(user)
+        return members.map{ it.login }.contains(user.login)
```

Comment:
This should be the bug fix - I found that my user was being returned as a team member (of development) from this call, but that the contains was returning false. I guess equality doesn't work on users, maybe some property was returned from one call but not the other?